lib/lib.misc.c

```diff
@@ -231,7 +231,8 @@ void daemonize(const char *_pidfile)
 
 	/* Close all other file descriptors. */
 	maxfd = sysconf(_SC_OPEN_MAX);
-	for(fd=3; fd<maxfd; fd++) close(fd);
+	/* Failure here is expected for some fds */
+	for(fd=3; fd<maxfd; fd++) if ( close(fd)!=0 ) continue;
```

Comment:
I'm not really a fan of this syntax, as it introduces additional control logic that effectively does nothing, but leaves you wondering why it's there. I'd just mark this as a "won't fix / false positive" in coverity as in cases like this there's really nothing to do to handle an error.

Reply:
I get your point, can I assume that the comment is also not needed? If so I'll close the PR.

Reply:
Indeed, that would be my suggestion.